agent/rpm/pbench-agent.spec.j2

```diff
@@ -95,26 +95,35 @@ fi
 %posttrans
 
 %files
-%defattr(775,pbench,pbench,775)
+%defattr(444,root,root,555)
+/%{installdir}/ansible
 /%{installdir}/config
-
 /%{installdir}/lib
-/%{installdir}/ansible
-/%{installdir}/base
 
+/%{installdir}/base
 /%{installdir}/VERSION
 /%{installdir}/SEQNO
 /%{installdir}/SHA1
 /%{installdir}/profile
 
-%ghost %attr(0400,pbench,pbench) /%{installdir}/id_rsa
+%config(noreplace) %attr(0664,root,root) /%{installdir}/config/pbench-agent.cfg
```

Comment:
Do we not still want the `%ghost` on the `id_rsa file?

Reply:
We don't have it in `main` where we still have support for the SSH operations.

Reply:
If this agent installation is their only copy of the server's `pbench` private key they're probably in trouble. Since we don't *install* one, we probably don't have to protect it against overwriting, either.

Reply:
Right.  RPM does not delete directories that have files in them.